test/jest/victory-bar/victory-bar.spec.js

```diff
@@ -0,0 +1,257 @@
+/*eslint-disable max-nested-callbacks */
+
+import React from "react";
+import { render, fireEvent, screen } from "@testing-library/react";
+import { range } from "lodash";
+import { VictoryBar, Bar } from "victory-bar";
+import { svgExpectations, getShapeUtils } from "../../svg-test-helper";
+
+describe("components/victory-bar", () => {
+  describe("default component rendering", () => {
+    it("accepts user props", () => {
+      const { container } = render(
+        <VictoryBar data-testid="victory-bar" aria-label="Chart" />
+      );
+
+      const svgNode = container.querySelector("svg");
+      expect(svgNode.getAttribute("data-testid")).toEqual("victory-bar");
```

Comment:
Now that we're using react testing library, we can change this to something like `expect(screen.getByTestId('victory-bar')).toBeDefined()`;